src/main/java/edu/cornell/kfs/pdp/CUPdpParameterConstants.java

```diff
@@ -4,7 +4,18 @@
 
     public static final String WARNING_INACTIVE_VENDOR_TO_EMAIL_ADDRESSES = "WARNING_INACTIVE_VENDOR_TO_EMAIL_ADDRESSES";
 
+    public static final String ACH_PERSONAL_CHECKING_TRANSACTION_CODE = "ACH_PERSONAL_CHECKING_TRANSACTION_CODE";
+    public static final String ACH_PERSONAL_SAVINGS_TRANSACTION_CODE = "ACH_PERSONAL_SAVINGS_TRANSACTION_CODE";
+    public static final String ACH_DIRECT_DEPOSIT_TRANSACTION_TYPE = "ACH_DIRECT_DEPOSIT_TRANSACTION_TYPE";
+    public static final String ACH_SECURE_FIELDS_ALLOWED_IN_ACCOUNT_EMAILS = "ACH_SECURE_FIELDS_ALLOWED_IN_ACCOUNT_EMAILS";

```

Comment:
In my opinion, it is not prudent to have this type of override ability as a user configurable application parameter; especially when that data can be sent out in emails.  The risks and ramifications of this type of data being accidentally exposed should this parameter be set incorrectly are too high. 
Was this an implementation decision or an end user requirement?  
Is there possibly another alternative?